satellite/console/service.go

```diff
@@ -117,7 +117,7 @@ func init() {
 // Config keeps track of core console service configuration parameters.
 type Config struct {
 	PasswordCost            int  `help:"password hashing cost (0=automatic)" testDefault:"4" default:"0"`
-	OpenRegistrationEnabled bool `help:"enable open registration" default:"false"`
+	OpenRegistrationEnabled bool `help:"enable open registration" default:"false" testDefault:"true"`
```

Comment:
This would enable it for all tests not just this one.

Reply:
Yes, I remember asking Jens why I wasn't able to create an account on testplanet but was able to on storj sim. He said open registration was enabled on storj sim by default and then showed me how to change the config to make it so I would be able to create an account on testplanet. He also told me to open a pull request to merge this with main, so in the future we wouldn't have to do this. Also, the reason why it needs to be enabled for other tests after this one is because we have tests regarding the onboarding wizard, which only shows up when you log in the first time with a new user.